.changeset/gentle-moles-punch.md

```diff
@@ -0,0 +1,6 @@
+---
+"@snapwp/query": minor
+"snapwp": minor
```

Comment:
Changesets is dumb, not semver. so long as we're bumping the third digit only it's a "patch"

```suggestion
"@snapwp/query": patch
"snapwp": patch
```